.cirrus.yml

```diff
@@ -316,16 +311,16 @@ special_testing_rootless_task:
 
     setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
     integration_test_script: '$SCRIPT_BASE/integration_test.sh |& ${TIMESTAMP}'
-    df_script: '${DFCMD}'
-    audit_log_script: 'cat /var/log/audit/audit.log || cat /var/log/kern.log'
-    journalctl_b_script: 'journalctl -b'
 
     on_failure:
         failed_master_script: '$CIRRUS_WORKING_DIR/$SCRIPT_BASE/notice_master_failure.sh'
-        # Job has already failed, don't fail again and miss collecting data
-        failed_df_script: '${DFCMD}'
-        failed_audit_log_script: 'cat /var/log/audit/audit.log || cat /var/log/kern.log || echo "Uh oh, cat audit.log failed"'
-        failed_journalctl_b_script: 'journalctl -b || echo "Uh oh, journalctl -b failed"'
+
+    always:  # Not using yaml alias in anticipation of specialization
```

Comment:
Feel free to change this one also: I need to think a lot harder about how to get `podman info`, but I do know that it doesn't belong here. Thanks!

Reply:
ok